src/main/java/io/ebeaninternal/server/deploy/DynamicPropertyAggregationFormula.java

```diff
@@ -49,15 +49,16 @@ public Object read(DataReader dataReader) throws SQLException {
 
   @Override
   public void load(SqlBeanLoad sqlBeanLoad) {
-
+    Object value;
     try {
-      Object value = scalarType.read(sqlBeanLoad.ctx().getDataReader());
-      if (asTarget != null) {
-        sqlBeanLoad.load(asTarget, value);
-      }
+      value = scalarType.read(sqlBeanLoad.ctx().getDataReader());
 
     } catch (Exception e) {
-      throw new PersistenceException("Error loading on " + fullName, e);
+      sqlBeanLoad.ctx().handleLoadError(null, asTarget, fullName, e);
+      return;
+    }
+    if (asTarget != null) {
```

Comment:
Why did this move?

Reply:
Do you mean, why I moved this out of the exception block?
I don't see when `sqlBeanLoad.load(asTarget, value);` can throw an exception that is worth to catch. (Of course, NPE, when the bean is null)